modules/subdyn/src/SD_FEM.f90

```diff
@@ -770,10 +807,12 @@ SUBROUTINE SetElementProperties(Init, p, ErrStat, ErrMsg)
    INTEGER                  :: I
    INTEGER                  :: N1, N2     ! starting node and ending node in the element
    INTEGER                  :: P1, P2     ! property set numbers for starting and ending nodes
+   INTEGER                  :: COSM_ind
    REAL(ReKi)               :: D1, D2, t1, t2, E, G, rho ! properties of a section
    REAL(FEKi)               :: DirCos(3, 3)              ! direction cosine matrices
+   REAL(FEKi)                       :: Dx_, Dy_, Dz_! distances between nodes
```

Comment:
Alignment appears to be off here. Make sure you don't use "Tab" in the code and only spaces. 

Anyway, I'm guessing these variables are temporary variables for debug and won't be needed.